inst/NORIC_local_monthly.Rmd

```diff
@@ -278,12 +278,9 @@ mst <- function(tab, type, cap, digs, align = NULL, fs = 8, lsd = FALSE) {
 }
 ```
 
-# `r params$hospitalName`
 
-# Hastegrad
 
```

Comment:
These and similar changes might break the export of this report. Needs to be tested!

Reply:
Level 1 and 2 heading are needed to make presentations (beamer and reveal) work as intended. After removal of level 2 unexpected things happen...

Reply:
> Level 1 and 2 heading are needed to make presentations (beamer and reveal) work as intended. After removal of level 2 unexpected things happen...


> Endringer i overskrifter (på nivå 2) gjør at eksport til presentasjonsformatene beamer (pdf) og reveal (html) gir feil "sidehåndtering". Enten må disse tas tilbake eller så fjernes brukervalgene for beamer og reveal (det siste er veldig enkelt å få til). Vet dere om disse formatene blir brukt mye?


Tror dette kan tas bort. Har bedt om bekreftelse..